apps/omg_eth/lib/omg_eth/encoding.ex

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 defmodule OMG.Eth.Encoding do
+  # TODO merge his with Utils.JsonRPC.Encoding
```

Comment:
the separation is intended here. One serves the encoding used to talk to the ethereum node, the other - talking to our RPC. One still might call the other for DRYness, since today these encoding coincide, but let's keep the invocations separate.